services/QuillLMS/app/controllers/pages_controller.rb

```diff
@@ -503,6 +503,12 @@ def locker
     @style_file = "#{ApplicationController::STAFF}.scss"
   end
 
+  def translations
+    return redirect_to profile_path if !staff?
```

Comment:
The linter might catch this, but I prefer `unless condition` over `if !condition`

Reply:
Ah, I copied it from the controller action above. I can change to `unless`, I like that better too.